frontend/.env.template

```
@@ -1,58 +1,12 @@
-PORT=8443
-
-# ---------------------------- #
-# REQUIRED LOCAL ENV VARIABLES #
-# ---------------------------- #
-# DEPLOYMENT_ENV=local
-# SITE_DOMAIN=localhost
-# SITE_URL=http://localhost:8443
-#/**
-#* We rely on the Nginx container running as `frontend_nginx`
-#* in the local compose stack to proxy requests. Therefore, the
-#* URL here is not for the Plausible container in the local stack,
-#* but the Nginx service, which then itself forwards the requests
-#* to the local Plausible instance.
-#*
-#* In production, the Nginx container is handling all requests
-#* made to the root URL (openverse.org), and is configured to
-#* forward Plausible requests to upstream Plausible.
-#*/
-# PLAUSIBLE_SITE_URL = http://localhost:50290
-
-# ---------------------------- #
-# OPTIONAL LOCAL ENV VARIABLES
-# ---------------------------- #
-### API settings
-
-# NUXT_PUBLIC_API_URL=http://127.17.0.1:8000/ # local dev API
-# NUXT_PUBLIC_API_URL=http://localhost:49153/ # talkback proxy
-# NUXT_PUBLIC_API_URL=https://api.openverse.org/# prod API
-
-# NUXT_API_CLIENT_ID=""
-# NUXT_API_CLIENT_SECRET=""
-
-# -------------------- #
-# PRODUCTION VARIABLES #
-# -------------------- #
-# DEPLOYMENT_ENV=production
-# NUXT_API_CLIENT_ID=""
-# NUXT_API_CLIENT_SECRET=""
-# NUXT_PUBLIC_API_URL=https://api.openverse.org/
-# NUXT_PUBLIC_SENTRY_DSN=""
-# NUXT_PUBLIC_SENTRY_RELEASE=""
-# SITE_URL=https://openverse.org
-# SITE_DOMAIN=openverse.org
-# PLAUSIBLE_SITE_URL=https://openverse.org
-
-# ----------------- #
-# STAGING VARIABLES #
-# ----------------- #
-# DEPLOYMENT_ENV=staging
-# NUXT_API_CLIENT_ID=""
-# NUXT_API_CLIENT_SECRET=""
-# NUXT_PUBLIC_API_URL=https://api-staging.openverse.org/
-# NUXT_PUBLIC_SENTRY_DSN=""
-# NUXT_PUBLIC_SENTRY_RELEASE=""
-# SITE_URL=https://staging.openverse.org
-# SITE_DOMAIN=staging.openverse.org
-# PLAUSIBLE_SITE_URL=https://staging.openverse.org
+NUXT_PUBLIC_DEPLOYMENT_ENV="local"
+NUXT_PUBLIC_API_URL="https://api.openverse.org/"
+NUXT_PUBLIC_SENTRY_DSN=""
+NUXT_PUBLIC_SENTRY_ENVIRONMENT="local"
+NUXT_PUBLIC_SENTRY_RELEASE="latest"
+NUXT_PUBLIC_PLAUSIBLE_ENABLED="true"
+NUXT_PUBLIC_PLAUSIBLE_DOMAIN="localhost"
+NUXT_PUBLIC_PLAUSIBLE_API_HOST="http://localhost:50290"
+NUXT_PUBLIC_AUTO_PAGEVIEWS="true"
+NUXT_PUBLIC_I18N_BASE_URL="http://localhost:8443"
+NUXT_PUBLIC_SITE_URL="http://localhost:8443" #can be read by Nuxt SEO, @see https://nuxtseo.com/site-config/getting-started/how-it-works#_6-runtime-config-and-environment-variables
+NUXT_PUBLIC_SITE_INDEXABLE="true"
```

```suggestion
NUXT_API_CLIENT_ID=
NUXT_API_CLIENT_SECRET=

NUXT_PUBLIC_API_URL="https://api.openverse.org/"
NUXT_PUBLIC_DEPLOYMENT_ENV="local"
NUXT_PUBLIC_I18N_BASE_URL="http://localhost:8443"

NUXT_PUBLIC_PLAUSIBLE_API_HOST="http://localhost:50290"
NUXT_PUBLIC_PLAUSIBLE_AUTO_PAGEVIEWS="true"
NUXT_PUBLIC_PLAUSIBLE_DOMAIN="localhost"
NUXT_PUBLIC_PLAUSIBLE_ENABLED="true"

NUXT_PUBLIC_PROVIDER_UPDATE_FREQUENCY=3600000
NUXT_PUBLIC_SAVED_SEARCH_COUNT=8

NUXT_PUBLIC_SENTRY_DSN=""
NUXT_PUBLIC_SENTRY_ENVIRONMENT="local"
NUXT_PUBLIC_SENTRY_RELEASE="latest"

NUXT_PUBLIC_SITE_INDEXABLE="true"
NUXT_PUBLIC_SITE_URL="http://localhost:8443"
```

I'd like to suggest some grouping for readibility and adding the non-public variables to the config as well.